### DISCUSSION TITLE
GOES 17

### DISCUSSION THREAD

**Discussion by hfrench** [JVital2013/vitality-goes#25] [Q&A] [answered]
Is this geared more for GOES 16 or will it work just as well with GOES 17?

Thanks Harry

**Comment by JVital2013** [comment, marked as answer]
It should work just as well for GOES-17. The default configs provided are based around GOES-16, but [I just added some sample configs under extra](https://github.com/JVital2013/vitality-goes/tree/main/extra) that demonstrates how it would work for GOES-17 (and eventually GOES-18).

Configs can be combined if you're receiving from more than one satellite as well. Full config documentation [can be found here](https://github.com/JVital2013/vitality-goes/blob/main/docs/config.md).

**Reply by hfrench** [reply]
Fantastic and thank you very much for your time and effort in this great software!!
